packages/web-worker/src/types.ts

```diff
@@ -0,0 +1,31 @@
+export type PromisifyModule<T> = {[K in keyof T]: PromisifyExport<T[K]>};
```

Comment:
I personally found [these _tests_](https://github.com/Shopify/quilt/pull/1084/files#diff-b4a026640b4aa57e3183d2c501452039R38) very helpful. @lemonmade, not sure if you want to keep them or not.